build.gradle

```diff
@@ -26,6 +21,12 @@ repositories {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.apache.commons', name: 'commons-lang3', version: "${project.ext.appacheCommonsVersion}"
+    compile 'org.apache.commons:commons-lang3' + "${appacheCommonsVersion}"
```

Comment:
Co to za kombinacje?
"org.apache.commons:commons-lang3:${appacheCommonsVersion}"